oak/server/rust/oak_loader/src/main.rs

```diff
@@ -78,7 +74,11 @@ fn read_file(filename: &str) -> Result<Vec<u8>, Box<dyn std::error::Error>> {
 }
 
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    simple_logger::init_by_env();
+    if cfg!(feature = "oak_debug") {
+        simple_logger::init_by_env();
+    } else {
+        eprintln!("No debugging output configured at build time");
```

Comment:
Does it mean that all logs will be suspended? Even `INFO`

Reply:
Yep, anything that could leak private information is compiled out.